test/integration/test-example-validation.js

```diff
@@ -14,17 +14,12 @@
  * limitations under the License.
  */
 
-
 import {loadPromise} from '../../src/event-helper';
 
-
-// TODO(@cramforce): Use local version. This is non-hermetic
-// and really bad. When the validator is open source we can
-// use it directly.
 if (!window.validatorLoad) {
   window.validatorLoad = (function() {
     const s = document.createElement('script');
-    s.src = 'https://cdn.ampproject.org/v0/validator.js';
+    s.src = '/validator/dist/validator_minified.js'; // Served by app.js.
```

Comment:
I could've sworn that the google3 style guide requires 2 spaces between `;` and `//` for inline comments, but I can't cite a source. I guess you're off the hook? :)